test/Microsoft.AspNet.Mvc.IntegrationTests/DictionaryModelBinderIntegrationTest.cs

```diff
@@ -129,14 +129,12 @@ public class DictionaryModelBinderIntegrationTest
             var entry = Assert.Single(modelState, kvp => kvp.Key == "parameter[low].Key").Value;
             Assert.Equal("key0", entry.AttemptedValue);
             Assert.Equal("key0", entry.RawValue);
-
-            // Key and Value are skipped if they have simple types.
-            Assert.Equal(ModelValidationState.Skipped, entry.ValidationState);
+            Assert.Equal(ModelValidationState.Valid, entry.ValidationState);
 
             entry = Assert.Single(modelState, kvp => kvp.Key == "parameter[low].Value").Value;
             Assert.Equal("10", entry.AttemptedValue);
             Assert.Equal("10", entry.RawValue);
-            Assert.Equal(ModelValidationState.Skipped, entry.ValidationState);
+            Assert.Equal(ModelValidationState.Valid, entry.ValidationState);
```

Comment:
It's coming back to me: A collection of simple types and a `KVP<,>` of simple types cannot have any applied validations. It's just useless to validate them. Excluding `KVP<,>` made perfect sense though I'm not sure why `IList<string>` wasn't automatically excluded as well.